Sources/App/Controllers/ClientController.swift

```diff
@@ -0,0 +1,52 @@
+struct ClientController {
```

Comment:
Why struct?

Reply:
Why we can need `class` here? No inheritance, no instance modifying. But I can make `class` if you wish, not a problem ))